packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,235 @@
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icon-system/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+
+<Callout variant="warning">
+  <CalloutText>
+    You can view and play with icons on <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list">our storybook icon list</Anchor>.
+    Our icon set is still limited, more will be adding over time. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/issues">file an icon request</Anchor> if you need an icon we
+    don't yet have. 
+  </CalloutText>
+</Callout>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+
+Icons are small graphical representation of a program or a function. They can be used to enhance the
+appearance of a web interface and break up the monotony of text. 
+
+Use an icon only when it provides additional clarity or is necessary to draw attention to a UI element.
+If an icon is used to indicate hierarchy of a UI element, all elements at the same hierarchy should also be paired with an icon.
+
+
+## Accessibility
+
+**Specify if an icon is decorative**
+
+Icons can either be **decorative** or **informative**. *"Decorative icons don’t add information to the content 
+of a page. The information provided by the image might already be given using adjacent text, or the image 
+might be included to make the website more visually attractive."*<sup><Anchor href="https://www.w3.org/WAI/tutorials/images/decorative/">w3.org</Anchor></sup> 
+
+Our icons require that the developer says if an icon is decorative by [providing the required property](/icon-system#making-an-icon-decorative-or-informative).
+
+**Give informative icons a title**
+
+A title is required so assistive technology can infer equal meaning as a sighted user would.
+
+<DoDont>
+  <Do title="Do" body="Use 'title' text that gives context and meaning to the icon." center>
+    <PlusIcon decorative={false} title="Add to cart" />
+    (hover the icon)
+  </Do>
+  <Dont title="Avoid" body="'title' text that only describes the icon image." center>
+    <PlusIcon decorative={false} title="Plus" />
+    (hover the icon)
+  </Dont>
+</DoDont>
+
+
+**Try pairing icons with text**
+
+Remember that different cultures may interpret the same image in different ways so despite our best intentions it is helpful to provide adjoining text.
+
+<Callout>
+  <CalloutText>
+    TODO: we should provide a better icon for this example when we have one. 
```

Comment:
Make this a public facing todo if you must. "We're working on a better example for this section!"